spec/support/config/test.yml

```diff
@@ -1,18 +1,18 @@
 name: amber_test_app
```

Comment:
Change the some of the contents of this file to ensure that the defaults are not used